sycl/test-e2e/FreeFunctionKernels/template_specialization.cpp

```diff
@@ -173,7 +173,10 @@ int main() {
   test_func_custom_type<A::B::C::TestClass>();
   test_func<F<float>, float>();
   test_func<F<uint32_t>, uint32_t>();
-  test_func<variadic_templated<double>, double>();
+  // Variadic template functions do not work with free function kernels. See
+  // CMPLRLLVM-69528.
+  // TODO: Uncomment the following line once the tracker is resolved.
+  //  test_func<variadic_templated<double>, double>();
```

Comment:
I'll leave review of this to @AlexeySachkov.

Reply:
> Is it a regression from this patch, or is it something previously known?

I believe it's previously known. The failures described by the linked issue has been happening since the test was created, it just wasn't caught by pre-commit since it only manifests with optimizations disabled.

Reply:
> > Is it a regression from this patch, or is it something previously known?
> 
> I believe it's previously known. The failure described by the tracker has been happening since the test was created, it just wasn't caught by pre-commit since it only manifests with optimizations disabled.

Ideally, such test should be disabled in a separate PR not to cause confusion, but I'm fine with the change since we do have a tracker to analyze the issue